FalmouthFalcons/iteration-3
# Ticket2/jq

Description:
-2. Seller can specify local delivery available

Number of Fixes:
-ticket 2, radio buttons are working properly and sending value to database

Related Ticket(s):
- Ticket 2

Problem to Solve:
- radio buttons are working properly and sending value to database

Proposed Changes:
-radio buttons are working properly and sending value to database

Expected Behavior:
- radio buttons are working properly and sending value to database

Steps to Test Solution
1) git checkout master
2) git fetch
3) git pull ticket2/jq
comment out windows gems
bundle install
rails db:migrate
rails db:seed

Documentation
[ ] There is new documentation in this pull request that must be reviewed..
[ ] I added documentation for any new classes/methods

Deploy Notes:
- Notes regarding deployment the contained body of work.
- These should note any db migrations, etc.
